### ISSUE TITLE
undefined method `configure' for AppleMusic:Module

### ISSUE THREAD

**Issue by Faisal-nfl** [yhirano55/apple_music#2]
I have added gem in rails 6 app.
& created a file in config/initializers named apple_music.rb with settings

```
AppleMusic.configure do |config|
  config.secret_key_path = ENV['APPLE_MUSIC_SECRET_KEY_PATH']
  config.team_id         = ENV['APPLE_MUSIC_TEAM_ID']
  config.music_id        = ENV['APPLE_MUSIC_MUSIC_ID']
  config.storefront      = 'us' 
end

```
but gives error saying 

undefined method `configure' for AppleMusic:Module

**Comment by murtyk** [comment]
yep, it doesn't work:

```ruby
[1] Vydia(main)> AppleMusic.configure do |config|
[1] Vydia(main)*   config.secret_key_path = './AuthKey_MUSIC_ID.p8' # or ENV['APPLE_MUSIC_SECRET_KEY_PATH']  
[1] Vydia(main)*   config.team_id         = 'YOUR TEAM_ID'          # or ENV['APPLE_MUSIC_TEAM_ID']  
[1] Vydia(main)*   config.music_id        = 'YOUR MUSIC_ID'         # or ENV['APPLE_MUSIC_MUSIC_ID']  
[1] Vydia(main)*   config.storefront      = 'jp'                    # or ENV['APPLE_MUSIC_STOREFRONT'] ('us' by default)  
[1] Vydia(main)* end  
NoMethodError: undefined method `configure' for AppleMusic:Module
from (pry):1:in `__pry__'
```

**Comment by wilg** [comment]
I think this is because a release has not been made, it does work on master: 

```
gem "apple_music", git: "git://github.com/yhirano55/apple_music.git"
```

**Comment by sjaveed** [comment]
@yhirano55 are you planning on publishing a release for this gem?

**Comment by yhirano55** [comment]
I'm sorry to have kept you waiting. I will publish it.

**Comment by yhirano55** [comment]
I've released v0.2.0.
